signalhooks/hooks.py

```diff
@@ -2,10 +2,22 @@
 import base64
 import boto3
 import requests
+import uuid
+import logging
 
 from django.core.serializers import serialize
 from django.contrib.contenttypes.models import ContentType
 
+logger = logging.getLogger(__name__)
+
+try:
+    from log_request_id import local
```

Comment:
Why this try except ?

Reply:
Because. We assume that when we use django_signalhooks, we have a log_request_id as a dependency, but maybe we didn't have